packages/common/android/src/main/java/com/reactnativevectoricons/common/VectorIconsModule.java

```diff
@@ -92,25 +78,12 @@ public String getImageForFontSync(String fontFamily, String glyph, double fontSi
     Canvas canvas = new Canvas(bitmap);
     canvas.drawText(glyph, offsetX, offsetY, paint);
 
-    try {
```

Comment:
this is a try-with-resources statement https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html

saves us from writing more code